src/pairing/pairing.tsx

```diff
@@ -24,9 +25,25 @@ export const Paring = () => {
       logger,
     })
 
-    connectorClient.setConnectionConfig(defaultConnectionConfig)
+    getExtensionOptions().map((options) => {
+      connectorClient.setConnectionConfig(
+        radixConnectConfig[options.radixConnectConfiguration],
+      )
+    })
 
     chrome.storage.onChanged.addListener((changes, area) => {
+      if (changes['options']) {
+        if (
+          changes['options'].newValue.radixConnectConfiguration !==
```

Comment:
Is it comparing two different objects?

Reply:
in options we only save name of the configuration, either `production` or `development`